receiver_guidance.md

```diff
@@ -0,0 +1,71 @@
+# Sending a Security Event Token as a receiver
+
+This page explains:
+
+- [the Shared Signals Framework (SSF)](#understanding-ssf-concepts) concepts you’ll need to understand as a receiver (#understanding-ssf-concepts)
+- [how a Security Event Token (SET) is used](#understanding-a-set)
+- [how to send a SET](#sending-a-set)
+
+## Understanding SSF concepts
+
+This application uses the following Shared Signals Framework (SSF) concepts:
+
+- transmitter - the entity responsible for broadcasting events
+- receiver - the entity responsible for receiving events
+
+A Relying Party (RP) is the transmitter of a Security Event Token (SET) and the SSF receiver endpoint is the receiver.
+
+## Understanding a SET
+
+A receiver issues a SET when a security subject changes state. For example, a security subject could be a user account or an HTTP session. When the SSF receiver endpoint receives a SET, the SSF receiver validates and interprets the SET. A transmitter and receiver can agree on an action upon receipt of a particular message.
+
+The SET format extends the JSON Web Token (JWT) format, which describes claims. The claims in a SET are outlined in [RFC 8417](https://datatracker.ietf.org/doc/html/rfc8417) and detail the:
+
+- security event that occurred
+- issuer
+- subject
+- intended audience of the event
+
+An example SET, used when an account has been disabled:
+
+```json
+{
+  "iss": "https://idp.example.com/",
+  "jti": "756E69717565206964656E746966696572",
+  "iat": 1508184845,
+  "aud": "636C69656E745F6964",
+  "events": {
+    "https://schemas.openid.net/secevent/risc/event-type/account-disabled": {
+      "subject": {
+        "subject_type": "iss-sub",
+        "iss": "https://idp.example.com/",
+        "sub": "7375626A656374"
+      },
+      "reason": "hijacking"
+    }
+  }
+}
+```
+
+The claims are signed to ensure integrity and authenticity. This format is known as JSON Web Signature (JWS) and is the most common representation of the JWT. JWS has the format: `{HEADER}.{PAYLOAD}.{SIGNATURE}`
```

Comment:
A JWT actually uses JWS for its signature. From the JWT spec:

JSON Web Token (JWT) is a compact, URL-safe means of representing claims to be transferred between two parties. The claims in a JWT are encoded as a JavaScript Object Notation (JSON) object that is used as the payload of a JSON Web Signature (JWS) structure or as the plaintext of a JSON Web Encryption (JWE) structure, enabling the claims to be digitally signed or MACed and/or encrypted.

So a JWT is actually a JWS structure with a JSON object as the payload.

Reply:
I was using this from the JWT RFC: "JWTs are always represented using the JWS Compact Serialization or the JWE Compact Serialization". I think I've now noticed the difference between JWS as the signature and JWS Compact Serialization as the format, which wasn't clear. 

I'll update this to: This format is known as JSON Web Signature (JWS) Compact Serialization format and is the most common representation of the JWT.